Update README to match 2025 state (#84)
elisa-tech/tsc: README.md

The mission of the ELISA project is to define and maintain a common set of elements, processes and tools that can be incorporated into Linux-based, safety-critical systems amenable to safety certification.

# Working Groups

Several working groups have been formed since the start of the ELISA project.
The working groups maintain their own repositories within the GitHub org for elisa-tech.

> Some working groups are representing horizontals like for tooling, architecture, features,

Reviewer: ```suggestion
Some working groups represent horizontals, such as for tooling, architecture and features,
```

Author: Implemented.

while others represent whole verticals like aerospace, automotive, and medical.

* [Safety Architecture](https://github.com/elisa-tech/Safety_Architecture_WG)
* [Linux Features](https://github.com/elisa-tech/wg-lfscs)
* [Engineering Process](https://github.com/elisa-tech/wg-osep)
* [Tools](https://github.com/elisa-tech/wg-tools)
  * [BASIL - requirements](https://github.com/elisa-tech/BASIL)
  * [Kernel Static Navigator](https://github.com/elisa-tech/ks-nav)
* [Systems](https://github.com/elisa-tech/wg-systems)
  * [Best practices Standard - lighthouse](https://github.com/elisa-tech/lighthouse-oss)
* [Aerospace](https://github.com/elisa-tech/wg-aerospace)
  * [Space Grade Linux - SGL](https://github.com/elisa-tech/sig-sgl)
    * [Yocto meta layer](https://github.com/elisa-tech/meta-sgl)
* [Automotive - covered in Systems WG](https://github.com/elisa-tech/wg-automotive)
  * [Yocto meta layer for AGL IC cluster](https://github.com/elisa-tech/meta-elisa)
* [Medical - paused](https://github.com/elisa-tech/wg-medical-devices)

More information can also be found at the official website under [Community->Working Groups](https://elisa.tech/community/working-groups/).

> Regular meetings are held and minutes are typically maintained in the working group's repository or GitHub wiki.

Reviewer 2: Maybe add´d a note that for TSC Meeting Minutes are stored in the wiki. 
```suggestion
Regular meetings are held and minutes are typically maintained in the working group's repository or GitHub wiki. The TSC meeting minutes can be found in the [wiki](https://github.com/elisa-tech/tsc/wiki) of this repository.
```

Author: Implemented.
